src/utils/schema-utils.js

```diff
@@ -76,6 +80,7 @@ export function getTypeInfo(schema) {
   info.constraint = constraint;
   info.html = JSON.stringify({
     type: info.type,
+    format: info.format,
```

Comment:
Do we need this in the `html` property if we use the `info.arrayItemFormat` property?

Reply:
yep, we need format of Integer, format of Array